### DISCUSSION TITLE
GB Cost Calculator and GB Calculator Open Window Automatically

### DISCUSSION THREAD

**Discussion by Vicarius23** [mainIine/foe-helfer-extension#2729] [Ideas]
My suggestion is to add the feature of opening the window automatically for GB Cost Calculator and GB Calculator. I find myself clicking the buttons constantly and closing to hide. Closing to hide is fine but you sure would save us all a lot of clicks by doing this. You know our hands are already tried from clicking. 🤣

**Comment by AlfonZ42** [comment]
Just making sure, if both options _Update GB Calculator for GBs of other players_ and the proposed _Open GB Calculator(s) automatically_ are enabled, does opening both calculator windows when visiting a friend's GB sound as an acceptable behaviour to you?

**Reply by Vicarius23** [reply]
Having them differentiate between my great buildings and others. Resulting in it opening the appropriate calculator. While being able to enable and disable each calculator automatically opening independent of each other. This is ideal situation.

However, anything is better than nothing. Both ways would save time, wrist movement, and clicks!
